app/src/main/java/zapsolutions/zap/fragments/SendBSDFragment.java

```diff
@@ -605,6 +588,13 @@ public void onClick(View v) {
         return view;
     }
 
+    @Override
+    public void onDestroyView() {
+        mHandler.removeCallbacksAndMessages(null);
```

Comment:
With this we remove all `Runnables` that are piled up and avoid them to run after view is destroyed.